### PR TITLE
windows build python >py35

### PR DESCRIPTION
**Pull request by ppolxda**
windows build python >py35
[https://ci.appveyor.com/project/ppolxda/plyvel](https://ci.appveyor.com/project/ppolxda/plyvel)

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,144 @@
+environment:
+  TWINE_USERNAME: bauerj
```
Comment (bauerj) [inline appveyor.yml]: Why is my username in there? 🔍 🤔

Comment (ppolxda) [reply appveyor.yml]: Because this commit is only testing your fork and fix some error.

**Comment by Bluscream** [comment]
Can you please readd the artifacts? https://ci.appveyor.com/project/ppolxda/plyvel/build/job/u1e76c600mouvt58/artifacts

**Comment by ppolxda** [comment]
> Can you please readd the artifacts? https://ci.appveyor.com/project/ppolxda/plyvel/build/job/u1e76c600mouvt58/artifacts

Ok, to see
https://github.com/ppolxda/plyvel/releases/tag/1.1.0.build-v0-win
https://ci.appveyor.com/project/ppolxda/plyvel

**Comment by Bluscream** [comment]
Thanks :)

**Pull request closed without merging**